### PR TITLE
fix length limitations of longtext and longblob

### LINKED ISSUE #8571: 关于LONGTEXT的最大size的相关记述
File: [/release-5.4/data-type-string.md](https://docs.pingcap.com/zh/tidb/stable/data-type-string)
关于数据类型 LONGTEXT 的最大size，按照我们文档的以下内容来看，以一个文字1个byte来看，看起来最大存储到4GB。
https://docs.pingcap.com/zh/tidb/stable/data-type-string#longtext-%E7%B1%BB%E5%9E%8B
------
LONGTEXT 类型
类似于 TEXT，区别在于最大列长度为 4,294,967,295。
------
但是我们实际上有单行的限制，最多只能存6M，实际上是存储不到4GB。
https://docs.pingcap.com/zh/tidb/stable/tidb-limitations#%E5%8D%95%E8%A1%8C%E7%9A%84%E9%99%90%E5%88%B6

所以麻烦在LONGTEXT 记载的相关界面上添加例如单行限制等tidb的存储的限制，以免客户误解。

### PR DESCRIPTION
**Pull request by djshow832**
### What is changed, added or deleted? (Required)

Fix #8571

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: data-type-string.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ MEDIUMTEXT [CHARACTER SET charset_name] [COLLATE collation_name]
 
 ### `LONGTEXT` 类型
 
-类似于 [`TEXT`](#text-类型)，区别在于最大列长度为 4,294,967,295。
+类似于 [`TEXT`](#text-类型)，区别在于最大列长度为 4,294,967,295。但由于 [TiDB 单列的限制](/tidb-limitations#单列的限制)，最大只能存储 6MB。
```
Comment (TomShawn) [inline data-type-string.md:74]:
```suggestion
类似于 [`TEXT`](#text-类型)，区别在于最大列长度为 4,294,967,295。但由于 [TiDB 单列的限制](/tidb-limitations.md#单列的限制)，TiDB 中单列存储最大不超过 6 MB。
```

File: data-type-string.md [hunk 2]
```diff
@@ -131,7 +131,7 @@ MEDIUMBLOB
 
 ### `LONGBLOB` 类型
 
-类似于 [`BLOB`](#blob-类型)，区别在于最大列长度为 4,294,967,295。
+类似于 [`BLOB`](#blob-类型)，区别在于最大列长度为 4,294,967,295。但由于 [TiDB 单列的限制](/tidb-limitations#单列的限制)，最大只能存储 6MB。
```
Comment (TomShawn) [inline data-type-string.md:134]:
```suggestion
类似于 [`BLOB`](#blob-类型)，区别在于最大列长度为 4,294,967,295。但由于 [TiDB 单列的限制](/tidb-limitations.md#单列的限制)，TiDB 中单列存储最大不超过 6 MB。。
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

File: data-type-string.md [hunk 3]
```diff
@@ -71,7 +71,7 @@ MEDIUMTEXT [CHARACTER SET charset_name] [COLLATE collation_name]
 
 ### `LONGTEXT` 类型
 
-类似于 [`TEXT`](#text-类型)，区别在于最大列长度为 4,294,967,295。
+类似于 [`TEXT`](#text-类型)，区别在于最大列长度为 4,294,967,295。但由于 [TiDB 单列的限制](/tidb-limitations.md#单列的限制)，TiDB 中单列存储最大不超过 6MB。
```
Comment (TomShawn) [inline data-type-string.md:74]:
```suggestion
类似于 [`TEXT`](#text-类型)，区别在于最大列长度为 4,294,967,295。但由于 [TiDB 单列的限制](/tidb-limitations.md#单列的限制)，TiDB 中单列存储最大不超过 6 MB。
```

File: data-type-string.md [hunk 4]
```diff
@@ -131,7 +131,7 @@ MEDIUMBLOB
 
 ### `LONGBLOB` 类型
 
-类似于 [`BLOB`](#blob-类型)，区别在于最大列长度为 4,294,967,295。
+类似于 [`BLOB`](#blob-类型)，区别在于最大列长度为 4,294,967,295。但由于 [TiDB 单列的限制](/tidb-limitations.md#单列的限制)，TiDB 中单列存储最大不超过 6MB。
```
Comment (TomShawn) [inline data-type-string.md:134]:
```suggestion
类似于 [`BLOB`](#blob-类型)，区别在于最大列长度为 4,294,967,295。但由于 [TiDB 单列的限制](/tidb-limitations.md#单列的限制)，TiDB 中单列存储最大不超过 6 MB。
```

**Review by TomShawn — APPROVED** [review]

**Review by tangenta — APPROVED** [review]

**Pull request merged**